### PR TITLE
feature/i18n-am-locale

### PR DESCRIPTION
**Pull request by IAmKio**
- Added AM (Amharic) locale.

### CODE REVIEW

File: src/configs/localeConfig.js [hunk 1]
```diff
@@ -38,6 +43,7 @@ export default {
   // pairs of language code and language name in native language
   supportedLanguages: {
     [DEFAULT_LANGUAGE_CODE]: DEFAULT_LANGUAGE,
+    am: 'Amharic',
```
Comment (vyvil) [inline src/configs/localeConfig.js:46]: Value (language name) should be in native language (for better UX) :)

**Review by vyvil — APPROVED** [review]
Well done! 🚀 
Just one note :)

File: src/configs/localeConfig.js (hunk 1 above)
Comment (IAmKio) [reply src/configs/localeConfig.js:46]: Thanks for pointing that out - I'll get this changed 👍🏾

Comment (IAmKio) [reply src/configs/localeConfig.js]: All done ✅

**Pull request merged**